sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ProgressReporter.scala

```diff
@@ -148,8 +148,12 @@ trait ProgressReporter extends Logging {
     currentTriggerEndTimestamp = triggerClock.getTimeMillis()
 
     val executionStats = extractExecutionStats(hasNewData)
-    val processingTimeSec =
-      (currentTriggerEndTimestamp - currentTriggerStartTimestamp).toDouble / MILLIS_PER_SECOND
+    val processingTimeMillis = currentTriggerEndTimestamp - currentTriggerStartTimestamp
+    val processingTimeSec = if (processingTimeMillis == 0) {
+      0.001d
```

Comment:
OK, I get it: pretend anything took at least 1 msec. Simpler might just be to write the line above as:
`val processingTimeMillis = math.max(1, currentTriggerEndTimestamp - currentTriggerStartTimestamp)` and not have an if statement here.